lib/getPostcssResult.js

```diff
@@ -1,13 +1,25 @@
 /* @flow */
 "use strict";
 
+const autoSyntax = require("postcss-html");
 const dynamicRequire = require("./dynamicRequire");
 const fs = require("fs");
-const lessSyntax = require("postcss-less");
 const path = require("path");
 const postcss = require("postcss");
-const scssSyntax = require("postcss-scss");
-const sugarssSyntax = require("sugarss");
+const safeParser = require("postcss-safe-parser");
+const sugarss = require("sugarss");
+const syntaxes /*: {
+  [syntaxName: string]: postcss$syntax,
+}*/ = {
+  // sass: require("postcss-sass"),
```

Comment:
Remove this, right now we don't support `sass`,  it should be added in separate PR